app/todos/models.py

```diff
@@ -1,8 +1,9 @@
 from app import db
 
+from app.lib.models import Base
 
-class Todo(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
+
+class Todo(Base):
```

Comment:
Same comments as above about using the declarative base, your mixing, and adding doc strings.